supervisor/gunicorn.conf

```diff
@@ -1,5 +1,5 @@
 [program:--DOMAIN_NAME--]
-command=envdir env bin/gunicorn --PROJECT--.wsgi -b unix:/var/www/%(program_name)s/gunicorn.sock
+command=envdir env bin/gunicorn --PROJECT--.wsgi -w 2 -b unix:/var/www/%(program_name)s/gunicorn.sock
```

Comment:
Just as one shouldn't use 1 letter variables in any other programming language, we should avoid it here. It's useful for throw-away code, but for scripts and configuration it's simply easier to read and understand with the long form.